server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const http = require('http');
+const PORT = process.env.PORT || 3000;
+const Router = require('./lib/router.js');
+const pokemonRouter = require('./router/pokemon-router.js');
+
+let router = new Router();
+
+pokemonRouter(router);
+
+
+//returns function with req,res
+const server = http.createServer(router.route());
+
+
+server.listen(PORT, () => {
+//   console.log(' .:XHHHHk.              db.   .;;.     dH  MX   0\n',
```

Comment:
Oh, haha, I just realized this is what comes up when you run your server. I never saw it because I just ran your tests to make sure you had things right. Nvm, do keep. :)